cypress/e2e/ScraperLiteEdition/APIInfo.cy.js

```diff
@@ -0,0 +1,97 @@
+import { SCRAPER_BASE_URL } from '../../resources/data';
+
+describe('Scraper(Lite Edition) APIs Info page', () => {
+    before(() => {
+        cy.visit(SCRAPER_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check for the "API Info" tab is present', () => {
+        cy.componentVisiblityCheck('#details-tab > .d-md-flex > span');
+    });
+
+    it('3. Test to click on the "API Info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('4. Test to click "Adanced Scraper API" link', () => {
+        cy.get('article > :nth-child(4) > a').click();
+        cy.document().should((doc) => {
+            expect(doc.readyState).to.equal('complete');
+        });
+        cy.assertPathname('/marketplace/adv_scraper-api');
+        cy.checkTextVisibility('Advanced Scraper API');
+        cy.go('back');
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('5. Test to check text "How to use CSS Selectors?" is present', () => {
+        cy.checkTextVisibility('How to use CSS Selectors?');
+    });
+
+    it('6. Test to check text "Image files scraping" is present', () => {
+        cy.checkTextVisibility('Image files scraping');
+    });
+
+    it('7. Test to check text "Rotating IP addresses" is present', () => {
+        cy.checkTextVisibility('Rotating IP addresses');
+    });
```

Comment:
write the test cases to check the ul elements in Rotating IP addresses
![image](https://github.com/user-attachments/assets/52f563a8-b478-4684-80ce-3079556fc59d)

Reply:
Added the ul list